test/images/porter/README.md

```diff
@@ -1,5 +1,9 @@
 This directory contains go source, Dockerfile and Makefile for making a test
 container which serves requested data on ports specified in ENV variables.
 
+The included localhost.crt is a PEM-encoded TLS cert with SAN IPs
+"127.0.0.1" and "[::1]", expiring at the last second of 2049 (the end
```

Comment:
they're going to get cert errors anyway, since the cert is self-signed... using the same TLS cert as the httptest TLS server was expedient. I added CERT_FILE/KEY_FILE envvars so a different cert/key can be mounted in and used if desired